en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -651,6 +748,64 @@ As these lists are not backed by the database, most of the filtering methods on
 this type. As such, an `UnsavedRelationList` should only be used for setting a relation before saving an object, not
 for displaying the objects contained in the relation.
 
+## Validating relations
+
+The [`RelationValidationService`](api:SilverStripe\Dev\Validation\RelationValidationService) can be used to check if your relations are set up according to best practices, and is very useful for debugging unexpected behaviour with relations. It is disabled by default.
+
+To enable this service, set the following yaml configuration, which will give you validation output every time you run `dev/build`.
+
+```yaml
+SilverStripe\Dev\Validation\RelationValidationService:
+  output_enabled: true
+```
+
+By default, this service only inspects relations for classes which have either no namespace or a namespace beginning with `App\`. You can declare your own namespaces by setting the `allow_rules` configuration:
+
+```yaml
+SilverStripe\Dev\Validation\RelationValidationService:
+  allow_rules:
+    # using the "app" key you can override the default "App" namespace
+    app: 'MyApp'
+    # you can add any namespace declarations with-or-without keys
+    - 'AnotherApp'
```

Comment:
> This mixed key/no-key notation looks really weird

That's fine, it's allowed to look weird, so long as it's correct and is accurately demonstrating what can be done.
In this case, it's demonstrating that you can choose to have keys or to omit them. Which is correct. This PR is not about making the docs show best practice, as mentioned in the description.

> having 'AnotherApp' as an example namespace is also weird because why would you have 2x "apps" in your project?

You wouldn't have two apps in the same project, but you also wouldn't declare "App" twice. This seemed like the lesser of two evils. But again, best practice isn't what this PR is about anyway.

Reply:
I've changed `AnotherApp` to `MyOrg`